packages/vscode-extension/src/commands/UpdateObject.ts

```diff
@@ -103,4 +99,8 @@ export default class UpdateObject extends BaseAction {
 			return this.unsuccessfulValidation(e.getErrorMessage());
 		}
 	}
+
+	private static isCustomRecord(scriptid: string): boolean {
```

Comment:
we don't really need that this method is static, it is not called for anywhere else.